packages/web/src/pages/saved-page/components/mobile/SavedPage.tsx

```diff
@@ -502,7 +502,7 @@ const PlaylistCardLineup = ({
               </div>
             </div>
           )}
-          <NewPlaylistButton />
+          <NewCollectionButton collectionType='playlist' />
```

Comment:
should we add these to the profile tabs for album and playlist like we have in web and native mobile?

Reply:
i checked with julian and he prefers having it in library on mobile